tests/unit/constraints/test_tabular.py

```diff
@@ -762,10 +762,8 @@ def test_transform(self):
         assert instance._uuids_to_combinations is not None
         expected_out_a = pd.Series(['a', 'b', 'c'], name='a')
         pd.testing.assert_series_equal(expected_out_a, out['a'])
-        try:
-            [uuid.UUID(u) for c, u in out['b#c'].items()]
-        except ValueError:
-            pytest.fail('ValueError')
+
+        [uuid.UUID(u) for _, u in out['b#c'].items()]
```

Comment:
It seems like this is just trying to check that the outputs are indeed all UUIDs, which is what they're supposed to be. Tbh I don't know if that is necessary for a unit test. At this point, the map of uuids to combinations should have already been made and we really just need to assert that the values are from that map like expected